src/QirRuntime/test/QIR-static/qir-test-arrays.qs

```diff
@@ -33,9 +35,9 @@ namespace Microsoft.Quantum.Testing.QIR
         if (dummy)
         {
```

Comment:
Similarly here.

Reply:
I will focus on _my_ changes for now.
However I will keep your code style recommendations in mind for the lines/functions/operations I touch. Thank you for sharing your knowledge. Please keep sharing, I appreciate that.